packages/indexer-common/src/errors.ts

```diff
@@ -112,6 +115,9 @@ export const INDEXER_ERROR_MESSAGES: Record<IndexerErrorCode, string> = {
   IE047: 'Failed to mark transfer as failed',
   IE048: 'Failed to withdraw query fees for allocation',
   IE049: 'Failed to clean up transfers for allocation',
+  IE050: 'Transaction reverted: gas limit likely hit (cumGasUsed > gasLimit * 0.9)',
```

Comment:
`cumGasUsed`? Why `likely`?

Reply:
Cumulative, I'll update that to not use the abbreviation. 

It's possible though unlikely that the transaction failed for another reason.